internal/wasm/sdk/opa/vm.go

```diff
@@ -113,8 +124,30 @@ func newVM(policy []byte, data []byte, memoryMin, memoryMax uint32) (*vm, error)
 		return nil, err
 	}
 
-	if data != nil {
-		if v.dataAddr, err = v.toRegoJSON(data, true); err != nil {
+	// Optimization for cloning a vm, if provided a parsed data memory buffer
```

Comment:
Nice! With this optimization we only have to parse the entire data set once as opposed to once per core? LGTM--though we'll want to get test coverage ASAP.

Reply:
Correct, we have one VM do the parse and then copy the parsed data section of memory into the other vm's.

> though we'll want to get test coverage ASAP
💯 💯